core-bundle/src/Migration/Version413/PageSubpagesLayoutMigration.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Migration\Version413;
+
+use Contao\CoreBundle\Migration\AbstractMigration;
+use Contao\CoreBundle\Migration\MigrationResult;
+use Doctrine\DBAL\Connection;
+
+/**
+ * @internal
+ */
+class PageSubpagesLayoutMigration extends AbstractMigration
+{
+    /**
+     * @var Connection
+     */
+    private $connection;
+
+    public function __construct(Connection $connection)
+    {
+        $this->connection = $connection;
+    }
+
+    public function shouldRun(): bool
+    {
+        $schemaManager = $this->connection->getSchemaManager();
+
+        if (!$schemaManager->tablesExist(['tl_page'])) {
+            return false;
+        }
+
+        $pageColumns = $schemaManager->listTableColumns('tl_page');
+
+        return !isset($pageColumns['subpageslayout']) && isset($pageColumns['layout'], $pageColumns['includeLayout']);
```

Comment:
Schema manager column names are all lowercase.

```suggestion
        return !isset($pageColumns['subpageslayout']) && isset($pageColumns['layout'], $pageColumns['includelayout']);
```

Reply:
Good point. However, prefilling the `subpagesLayout` will not be necessary with the current concept of `foreignOptions` and this migration would be removed. 

I will keep this in mind for possibly adding a migration to adjust the access rights for users and user groups.